assets/models/system/bert-base-cased/description.md

```diff
@@ -31,12 +31,14 @@ Fill Mask|Fill Mask|<a href="https://huggingface.co/datasets/rcds/wikipedia-for-
 #### Sample input
 ```json
 {
-    "inputs": {
```

Comment:
is it expected to have bert-base-cased changes in this PR as well as in: https://github.com/Azure/azureml-assets/pull/1106/files